src/nanoserver/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+RUN pwsh -ExecutionPolicy Unrestricted -c "Invoke-WebRequest -OutFile .\python-3.7.3-nuget-amd64.zip https://www.nuget.org/api/v2/package/python/3.7.3 ; Expand-Archive python-3.7.3-nuget-amd64.zip -DestinationPath c:\PythonTemp -Force" \
+    && del /q .\python-3.7.3-nuget-amd64.zip \
+	&& md c:\Python \
+	&& xcopy /s c:\PythonTemp\tools C:\Python \
+	&& rd /s /q c:\PythonTemp 	
+
+USER ContainerAdministrator 
+
+RUN setx /M PATH "%PATH%;C:\Program Files\PowerShell;C:\Python;C:\python\scripts" \
+ && setx /M PYTHONPATH "C:\Python\Lib;C:\Python\DLLs;" 
+
+RUN mkdir c:\Work \
```

Comment:
What I meant by my previous comment was that it shouldn't be necessary to explicitly call `mkdir c:\Work`.  `WORKDIR Work` will create the directory if it doesn't already exist.